src/routers/sessions.py

```
@@ -250,6 +250,31 @@ async def parse_stream():
         )
 
 
+@router.get(
+    "/{session_id}/deriver/status",
+    response_model=schemas.DeriverStatus,
+    dependencies=[
+        Depends(
+            require_auth(app_id="app_id", user_id="user_id", session_id="session_id")
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)